boa/environment.py

```diff
@@ -478,9 +480,29 @@ def enable_fast_mode(self, flag: bool = True):
         else:
             unpatch_pyevm_state_object(self.vm.state)
 
-    def fork(self, url, reset_traces=True, **kwargs):
-        kwargs["url"] = url
-        AccountDBFork._rpc_init_kwargs = kwargs
+    def fork(
+        self, url=None, rpc=None, reset_traces=True, block_identifier="safe", **kwargs
+    ):
+        """
+        Fork the environment to a local chain.
+        :param url: URL of the chain to fork from
+        :param rpc: RPC to fork from
+        :param reset_traces: Reset the traces
+        :param block_identifier: Block identifier to fork from
+        :param kwargs: Additional arguments for the RPC
+        """
+        if bool(rpc) == bool(url):
```

Comment:
let's split this into another function like `fork_rpc()`, i think it's cleaner from a UX perspective

Reply:
Done